text/0000-clarify-api-namespace.md

```diff
@@ -0,0 +1,41 @@
+- Start Date: 2015-08-10
+- RFC PR: (leave this empty)
+- Refract Issue: (leave this empty)
+
+# Summary
+Clarify API description namespace and iron out issues which rose during API Blueprint serialization to API description namespace.
+
+# Motivation
+The purpose of this document is to clear inconsistencies in the format and make it easier to build tools around it.
+
+# Detailed design
+
+## Data Structure
+Since `Data Structure` inheriting from `Data Structure Element` of DataStructure Correct the `Data Structure` to the following:
```

Comment:
This sentence does not make much sense as such. Also I do not think "DataStructure" is used anywhere else.